Digipost/POSAttachment.h

```diff
@@ -47,6 +47,8 @@ extern NSString *const kAuthenticationLevelPassword;
 @property (nonatomic, retain) POSDocument *document;
 @property (nonatomic, retain) POSInvoice *invoice;
 @property (nonatomic, retain) NSString *origin;
+@property (nonatomic, retain) NSNumber *endToEndEncrypted;
+@property (nonatomic, retain) NSNumber * userKeyEncrypted;
```

Comment:
Yes, and i've turned off clang formatting when doing these mergings, can do a commit with fix for it, to make it look nicer.